src/plugins/intel_gpu/src/plugin/transformations/rms_fusion.cpp

```diff
@@ -0,0 +1,121 @@
+
+    // ReduceMean(x^2,axes)
+    auto mean_axes = wrap_type<ov::op::v0::Constant>(constant_value(-1));
+    auto mean = wrap_type<ov::op::v1::ReduceMean>({power, mean_axes});
+
+    // ReduceMean(x^2,axes)+eps
+    auto eps = wrap_type<ov::op::v0::Constant>();
+    auto add_eps = wrap_type<ov::op::v1::Add>({mean, eps});
+
+    // Sqrt(ReduceMean(x^2,axes)+eps)
+    auto sqrt = wrap_type<ov::op::v0::Sqrt>({add_eps});
+
+    // 1/Sqrt(ReduceMean(x^2,axes)+eps)
+    auto const_div = wrap_type<ov::op::v0::Constant>(constant_value(-1));
+    auto div = wrap_type<ov::op::v1::Power>({sqrt, const_div});
+
+    // x * 1/Sqrt(ReduceMean(x^2,axes)+eps)
+    auto mul1 = wrap_type<ov::op::v1::Multiply>({x, div});
+
+    // x * 1/Sqrt(ReduceMean(x^2,axes)+eps) * gamma
+    auto gamma = wrap_type<ov::op::v0::Constant>(type_matches(element::f32));
+    auto mul2 = wrap_type<ov::op::v1::Multiply>({gamma, mul1});
+
+    // compress RMS result
+    auto comp = wrap_type<ov::op::v0::Convert>({mul2});
+
+    ov::matcher_pass_callback callback = [=](ov::pass::pattern::Matcher& m) {
+        const auto& pattern_map = m.get_pattern_value_map();
+        auto x_output = pattern_map.at(x);
+
+        auto const_eps_node =
+            std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_map.at(eps).get_node_shared_ptr());
+        float eps_value;
+        if (!ov::op::util::get_single_value(const_eps_node, eps_value)) {
+            return false;
+        }
+
+        auto gamma_node =
+             std::dynamic_pointer_cast<ov::op::v0::Constant>(pattern_map.at(gamma).get_node_shared_ptr());
+        if (!gamma_node) {
+            return false;
+        }
+
+        auto comp_node =
+            std::dynamic_pointer_cast<ov::op::v0::Convert>(pattern_map.at(comp).get_node_shared_ptr());
+        if (!comp_node) {
+            return false;
+        }
+
+        ov::NodeVector nodes_to_copy_info({pattern_map.at(power).get_node_shared_ptr(),
+                                           pattern_map.at(mean).get_node_shared_ptr(),
+                                           pattern_map.at(add_eps).get_node_shared_ptr(),
+                                           pattern_map.at(sqrt).get_node_shared_ptr(),
+                                           pattern_map.at(div).get_node_shared_ptr(),
+                                           pattern_map.at(mul1).get_node_shared_ptr(),
+                                           pattern_map.at(mul2).get_node_shared_ptr(),
+                                           pattern_map.at(comp).get_node_shared_ptr()});
+
+        auto rms = std::make_shared<op::RMS>(x_output,
+                                             gamma_node,
+                                             eps_value,
+                                             comp_node->get_destination_type());
+        rms->set_friendly_name(m.get_match_root()->get_friendly_name());
+        ov::copy_runtime_info(nodes_to_copy_info, rms);
```

Comment:
I think you can use `ov::copy_runtime_info(m.get_matched_nodes(), rms);` here and remove `nodes_to_copy_info`

Reply:
updated it as suggested